src/main/resources/application.yml

```diff
@@ -68,6 +68,10 @@ interventions-ui:
   locations:
     sent-referral: "/referrals/{id}/confirmation"
 
+community-api:
```

Comment:
this should match with the toplevel config section name in `application-local.yml`, which is `communityapi`

Reply:
Changed to community-api